napari_brightness_contrast/_dock_widget.py

```diff
@@ -241,7 +237,7 @@ def _auto_percentiles(self):
 
     def _set_full_range(self):
         for layer in self.selected_image_layers():
-            layer.contrast_limits = min_max(layer.data)
+            layer.contrast_limits = min_max(layer._data_view)
```

Comment:
I'm wondering if this is a good idea. Can you explain why you're accessing a private member here? Is it safe to do so?

Reply:
The private member is the one containing the current frame, so it does not require using the whole n-dimension image for the brightness/contrast. I use the private because I could not find a better approach yet